src/manifest.json

```diff
@@ -6,7 +6,7 @@
     "icons": {
         "128": "assets/icon-128.png"
     },
-    "content_security_policy": "script-src 'self'; object-src 'self'",
+    "content_security_policy": "script-src 'self' 'unsafe-eval'; object-src 'self'",
```

Comment:
this was actually removed for a reason, you need to revert that locally for development, but should not commit it

Reply:
Oops! Sorry for the misunderstanding. Will avoid in future.